css/styles.css

```diff
@@ -107,3 +108,77 @@ ul {
 .fas.fa-pen {
   color: #562b81;
 }
+
+.delete {
+  color: red;
+  position: relative;
+  bottom: 1.5rem;
+  width: fit-content;
+}
+
+.delete.clicked {
+  opacity: 0.5;
+}
+
+.modalContainer {
+  border: 1px solid black;
+  text-align: center;
+  height: 8rem;
+  width: 75%;
+  position: fixed;
+  right: 12%;
+  top: 50%;
+  background-color: white;
+  border-radius: 0.25rem;
+}
+
+.cancelButton {
+  float: left;
+  margin-left: 1.5rem;
+  background-color: rgb(177, 177, 177);
+  padding: 0.35rem 1rem;
+  border: 1px solid rgb(177, 177, 177);
+  border-radius: 0.25rem;
+  margin-top: 1rem;
+  font-size: 1rem;
+}
+
+.confirmButton {
+  float: right;
+  margin-right: 1.5rem;
+  background-color: red;
+  padding: 0.35rem 0.75rem;
+  border: 1px solid red;
+  border-radius: 0.25rem;
+  color: white;
+  margin-top: 1rem;
+  font-size: 1rem;
+}
+
+@media screen and (min-width: 768px) {
+  .modalContainer {
+    width: 25rem;
+    top: 50%;
+    left: 50%;
+    margin-top: -3.5rem;
+    margin-left: -12.5rem;
```

Comment:
WOAH that is a lot of margin adjustment, and especially bad, since it is a negative value. Why is this necessary? If you want to use negative amrgin, that probably means that another element just needs a smaller margin. Is this to center the element? If so, you can use `transform: translate(-50%, -50%)`. The problem here is this:
```
    top: 50%;
    left: 50%;
```
This code will center the _top left corner_ of the element, not the center, so it ends up being off-center. The line I gave you shifts the point that is being centered to the center of the element, by moving the origin point backwards by 50% of the height and width of the element. Now the top left corner won't be centered, it will be the center point of the element.